ai-assistant-product/README.md

```diff
@@ -1 +1,671 @@
 # ai-assistant
+
+<todo>
+
+## Demo
+
+### Demo project
+
+In the **ai-assistant-demo** Ivy project, we created demo assistants which can help you understand and develop your own AI assistant more effeciency.
+
+> [!IMPORTANT]
+> This demo project will create Ivy users and roles and override the main variables of the AI Assistant. Therefore, it is HIGHLY RECOMMENDED to run it on an Ivy engine in demo mode to protect your data.
+>
+> If you choose to run this demo in a production environment, please ensure you back up all Ivy variables related to AI Assistant and restore them once the demo is complete.
+
+#### Complex Demo
+
+**Use case**
+
+User can plan a software project with AI Assistant. The AI Assistant enables users to efficiently plan a software project by organizing information into a structured format. It can:
+
+- Create the project and persist it in the database
```

Comment:
start the listing not with capital letters (as these are not complete sentences): 


- „create the project and persist it in the database“
- „assist in finding and selecting suitable team members for the project“
- „help users prepare for the project kick-off meeting“
- „create and assign tasks to invite team members to the meeting“